packages/lib/universal/get-feature-flag.ts

```
@@ -0,0 +1,108 @@
+ * Evaluate whether a flag is enabled for the current user.
+ *
+ * @param flag The flag to evaluate.
+ * @param options See `GetFlagOptions`.
+ * @returns Whether the flag is enabled, or the variant value of the flag.
+ */
+export const getFlag = async (
+  flag: string,
+  options?: GetFlagOptions,
+): Promise<TFeatureFlagValue> => {
+  const requestHeaders = options?.requestHeaders ?? {};
+  delete requestHeaders['content-length'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)